### DISCUSSION TITLE
is win64 version bit a malware

### DISCUSSION THREAD

**Discussion by Joel-Nickson** [42wim/matterbridge#1404] [Help / Q&A] [answered]
was trying to download matterbridge and found that all of your files are above 40 mb except the win 64 bit version which is only 800bytes ,
it seems suspicious and even the windows thinks that its a malware after download ...
same with the 32 bit version (thinks it could harm the pc)

**Comment by 42wim** [comment, marked as answer]
That's why there is a checksum: https://github.com/42wim/matterbridge/releases/download/v1.22.0/checksums.txt 
The win64-bit is also 50MB

![image](https://user-images.githubusercontent.com/1810977/108181204-ebd13c00-7107-11eb-8aec-ebf2a6116400.png)

Your download probably failed, redownload
